apps/nextjs/src/app/[locale]/boards/(content)/_header-actions.tsx

```diff
@@ -105,6 +121,10 @@ const AddMenu = () => {
           {t("item.action.create")}
         </Menu.Item>
 
+        <Menu.Item leftSection={<IconApps size={20} />} onClick={handleSelectApp}>
```

Comment:
I would suggest that we change the icon for items and use the same icon for apps as we have in management. There we use `IconBox` for apps. Maybe we can use `IconResize` or something similar for items